index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Porfolio</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="./Images/all.min.css">
+</head>
+<body>
+  <header class="container">
+    <a href="#" class="welcome">My Logo</a>
+    <button type="button"><img class="icon" src="./eclips/menuicon.png" alt="icon"></button>
+  </header>
+  <img class="eclip11" src="./eclips/Ellipse11.png" alt="Eclips">
+  <img class="eclip8" src="./eclips/Ellipse8.png" alt="Eclips">
+  <img class="eclip5" src="./eclips/Ellipse5.png" alt="Eclips">
+  <img class="eclip3" src="./eclips/Ellipse3(1).png" alt="Eclips">
+  <img class="rect4" src="./eclips/Rectangle4.png" alt="small-rectangle">
```

Comment:
As mentioned by the previous reviewer, since these background images are for decoration , Kindly use CSS selectors to add them not `img` tag. 👍 
For example: inside your CSS file you can use `background-image:url(imageurl)`
This way is more professional 
Kindly refer to this [link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more information
I hope you understand the point now :) 👍